Dockerfile.ubi

```diff
@@ -183,6 +183,11 @@ RUN --mount=type=cache,target=/root/.cache/pip \
 ENV HF_HUB_OFFLINE=1 \
     PORT=8000 \
     HOME=/home/vllm \
+    # As an optimization, vLLM disables logprobs when using spec decoding by
+    # default, but this would be unexpected to users of a hosted model that
+    # happens to have spec decoding
+    # see: https://github.com/vllm-project/vllm/pull/6485
+    DISABLE_LOGPROBS_DURING_SPEC_DECODING=false \
```

Comment:
This will work for the adapter entrypoint but not for the upstream one, as `EnvVarArgumentParser` is only used in the adapter.

How about moving this in the `vllm-grpc-adapter` stage below to make it explicit that this is only set when using the adapter?

We should also move any other adapter-only env vars below and/or convert env vars to command line arguments when this also needs to apply to standard deployments

Reply:
Not for this, but we could also consider offering the `EnvVarArgumentParser` to upstream

Reply:
👍  Pushed a change to move the env vars.

Reply:
> Not for this, but we could also consider offering the EnvVarArgumentParser to upstream

Strongly agree, it's a nice improvement.